common/services/prismic/parsers.js

```diff
@@ -527,6 +530,26 @@ function getWeight(weight: ?string): ?Weight {
   }
 }
 
+function parseMediaObjectLinks(
+  fragment: PrismicFragment[]
+): Array<MediaObjectType> {
+  // make sure linked type is media_object
+  const filteredMediaObject = fragment.filter(mediaObject => {
+    if (mediaObject.content?.type === 'media_object') {
+      return mediaObject;
+    }
+  });
```

Comment:
Filter should be returning a `boolean`. 

```suggestion
  const filteredMediaObject = fragment.filter(mediaObject => mediaObject.content?.type === 'media_object');
```

Reply:
> Assuming this has to be added to all body blocks?

yes in prismic is will be a slice which gets inserted into the body on a page type.